cicd/3-app/javabuilder/config/test.config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Parameters": {
+    "SubDomainName": "javabuilder-test",
+    "BaseDomainName": "code.org",
+    "BaseDomainNameHostedZonedID": "Z2LCOI49SCXUGU",
+    "ProvisionedConcurrentExecutions": "1",
+    "ReservedConcurrentExecutions": "3",
+    "LimitPerHour": "15",
```

Comment:
Is this where  integration tests are run? If so I think we should increase these limits to avoid noisy errors. Maybe 40 per hour, 100 per day?